sympy/physics/control/control_plots.py

```diff
@@ -863,18 +865,35 @@ def bode_phase_numerical_data(system, initial_exp=-5, final_exp=5, freq_unit='ra
     x, y = LineOver1DRangeSeries(phase,
         (_w, 10**initial_exp, 10**final_exp), xscale='log', **kwargs).get_points()
 
+    if phase_unwrap:
```

Comment:
Will the result always be continuous after this or is it possible that a portion has jumped by more than one unit?

Reply:
> Will the result always be continuous after this or is it possible that a portion has jumped by more than one unit?

No.
I've tried this out with bode plot examples documented in `MATLAB`. If you have any particular Transfer functions in mind, you can share them here. Thanks !